pyproject.toml

```diff
@@ -67,7 +67,7 @@ feflow = [
 ]
 
 [tool.versioningit]
-default-version = "1+unknown"
+default-version = "0.1.0+unknown"
```

Comment:
Not sure what this does; we leave it as "1+unknown" in `alchemiscale`. @mikemhenry?